Model/DutyCalculator.php

```diff
@@ -261,36 +280,35 @@ public function fillOutQuoteAndSessionUsingFeeReturned($duty, $address, $apply,
         $quote = $this->checkoutSession->getQuote();
         $duty_adjusted = $this->priceCurrency->round($duty); //copied over pre-existing code
         //but are we supposed to round always?
-        //should not that be based on corresponding admin setting?
+        //should not that be based on corresponding admin setting? or what is coming back from REACH API
         $this->checkoutSession->setReachDuty($duty_adjusted);
 
+        $baseDuty = $this->getBaseDuty($duty_adjusted);
+
+        $this->checkoutSession->setBaseReachDuty($baseDuty);
+
+        $this->checkoutSession->setDhlQuoteId($response['quoteId']);
+        //$this->checkoutSession->setReachDuty($duty_adjusted);
```

Comment:
this comment can be removed since this same call is being made on line 284 (just a few lines above) `$this->checkoutSession->setReachDuty($duty_adjusted);`

Reply:
Right. Thanks!

Reply:
commit 971e7cc32fde044bdeb84a06f3d5308baf9713cb 
Thanks!